addons/l10n_ec/data/account.account.template.csv

```diff
@@ -36,7 +36,11 @@ ec_purchase_vat_goods_imports,11050104,IVA pagado en importacion de bienes,FALSE
 ec_purchase_vat_assets_imports,11050105,IVA pagado en importacion de activos fijos,FALSE,asset_current,l10n_ec_ifrs
```

Comment:
please include the accounts for the plastic bottles so that we have one single PR modifying the chart of accounts

Reply:
Ok. It's ready.